srcpkgs/howl/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'howl'
+pkgname=howl
+version=0.6
+revision=1
+archs="i686 x86_64 x86_64-musl"
```

Comment:
Why doesn't this run on arch? If it is a cross-building issue, it should be marked `nocross` instead of limiting `archs`.

Reply:
I'm not sure. I've posted about it [earlier](https://github.com/void-linux/void-packages/pull/21717#issuecomment-625227438) with a build log. I'm not accustomed to cross building, so have no clue why this is happening.

But I've removed `archs` and added `nocross` with link to failed build log.